src/webui/www/private/views/about.html

```diff
@@ -676,6 +676,7 @@ <h3><a name="howto"></a><a name="SEC4" href="#TOC4">How to Apply These Terms to
             <td><span id="zlibVersion"></span></td>
         </tr>
     </table>
+    <p>QBT_TR(The free)QBT_TR[CONTEXT=AboutDialog] <a href="https://db-ip.com/db/download/ip-to-country-lite" target="_blank" rel="noopener noreferrer">QBT_TR(IP to Country Lite database)QBT_TR[CONTEXT=AboutDialog]</a> QBT_TR(by)QBT_TR[CONTEXT=AboutDialog] <a href="https://db-ip.com/" target="_blank" rel="noopener noreferrer">DB-IP</a> QBT_TR(is used for resolving the countries of peers. The database is licensed under the)QBT_TR[CONTEXT=AboutDialog] <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noopener noreferrer">QBT_TR(Creative Commons Attribution 4.0 International License)QBT_TR[CONTEXT=AboutDialog]</a>.</p>
```

Comment:
The paragraph presented to translators is very broken. They see: "The free", "IP to Country Lite database", "by"... how one is able to work with it?
An easy and acceptable way for me is just strip all html links and present all text as a string for translation.